calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -204,7 +209,6 @@ def to_json(self):
         json_dict = {"state": self.state,
                      "name": self.name,
                      "mac": self.mac,
-                     "container:if_name": self.if_name,
```

Comment:
I don't think so - but perhaps I'm wrong.

This used to be used in the reinstating endpoint which was a feature of powerstrip... we don't reference that any more.